slackminion/plugin/manager.py

```diff
@@ -111,10 +111,14 @@ def load_state(self):
         if self.state_handler is None:
             self.log.warn("Unable to load state, no handler registered")
             return
+        try:
+            state_str = self.state_handler.load_state()
```

Comment:
Previously, an exception was logged if the state file was not found. Since this is a normal case when starting up the app for the first time, it was a bit confusing to see a stacktrace. Now this case is handled and considered "normal". All other exceptions are still logged.